smart-contracts/for-eth-devs/accounts.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Accounts in Asset Hub Smart Contracts
+title: Accounts in Polkadot Hub Smart Contracts
 description: Bridges Ethereum's 20-byte addresses with Polkadot's 32-byte accounts, enabling seamless interaction while maintaining compatibility with Ethereum tooling.
 categories: Basics, Polkadot Protocol
 ---
 
-# Accounts on Asset Hub Smart Contracts
+# Accounts on the Polkadot Hub Smart Contracts
 
 ## Introduction
 
-Asset Hub natively utilizes Polkadot's 32-byte account system while providing interoperability with Ethereum's 20-byte addresses through an automatic conversion system. When interacting with smart contracts:
+The Polkadot Hub natively utilizes Polkadot's 32-byte account system while providing interoperability with Ethereum's 20-byte addresses through an automatic conversion system. When interacting with smart contracts:
 
 - Ethereum-compatible wallets (like MetaMask) can use their familiar 20-byte addresses.
 - Polkadot accounts continue using their native 32-byte format.
-- The Asset Hub chain automatically handles conversion between the two formats behind the scenes:
+- The Polkadot Hub chain automatically handles conversion between the two formats behind the scenes:
```

Comment:
This makes more sense to have "The" in front of "Polkadot Hub" cause you're referring to the chain explicitly